src/shared/dissect-image.c

```diff
@@ -2110,45 +2123,36 @@ int dissected_image_mount(
         if (r < 0)
                 return r;
 
-        xbootldr_mounted = mount_partition(PARTITION_XBOOTLDR, m->partitions + PARTITION_XBOOTLDR, where, "/boot", uid_shift, uid_range, flags);
-        if (xbootldr_mounted < 0)
-                return xbootldr_mounted;
+        int slash_boot_is_available = -ENODATA;
```

Comment:
if you initialize this in the very next line i'd just drop the initialization here